schema.ts

```diff
@@ -56,19 +56,16 @@ export const StringHintValueTypes = [
   ValueHintType.Markdown,
   ValueHintType.Url,
 ] as const;
-export const NumberHintValueTypes = [
+export const SimpleNumberHintValueTypes = [
```

Comment:
What's the distinction of simple hint types vs others? Date/Time/DateTime/Percent all have extra schema fields too.

Reply:
oops, sorry for the messiness, will fix this